bench/run-1yr.sh

```
@@ -0,0 +1,43 @@
+  for name in $NAMES; do
+    eval "pk=\$PK_${name}"
+    logfile="$LOGDIR/${name}_s${session}.log"
+
+    # Skip if already completed
+    if [ -f "$logfile" ] && grep -q "Phase 2" "$logfile" 2>/dev/null; then
+      echo "SKIP (already done): ${name}_s${session}"
+      continue
+    fi
+
+    echo "=== Session $session: $name (1yr) ==="
+    deno task bench "$pk" --algorithms "$ALGOS" $COMMON > "$logfile" 2>&1 || true
```

Shellcheck (0.11.0)

[warning] pk is referenced but not assigned.

(SC2154)